implicits/src/com/rallyhealth/weepickle/v1/implicits/internal/Macros.scala

```diff
@@ -390,9 +390,17 @@ object Macros {
         """
 
         /**
-          * @see [[shouldDropDefault()]]
+          * Transforming the argument to itself applies any logic that the client may have provided
+          * in the implicit definition of From and/or To for the data type. This is important as this
+          * logic may transform the value into a default value which needs to be dropped.
+          *
+          * @see [[Argument.omitDefault]].
           */
-        if (arg.writingCheckDefault) q"""if (v.${TermName(arg.raw)} != ${arg.default}) $snippet"""
+        if (arg.writingCheckDefault)
+          q"""
+             val fromToArg = implicitly[${c.prefix}.FromTo[${arg.argType}]]
```

Comment:
Is there a way to opt-out if we want to define a `From[CaseClass]` with default fields, but don't have a `To[Field]`?

Reply:
See above. And I'm totally open to other suggestions. My main objective is to pass the new tests at a reasonable price.